dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/core/process_definition.py

```diff
@@ -414,12 +414,8 @@ def submit(self) -> int:
         )
         if len(self.resource_list) > 0:
             for res in self.resource_list:
-                gateway.entry_point.createOrUpdateResource(
-                    self._user,
-                    res.name,
-                    res.description,
-                    res.content,
-                )
+                res.user_name = self._user
```

Comment:
I have a question here, why can we set `res.user_name` here? `self.resource_list` is `optional[List]`. how about users pass `[1, 3, 4]` to it?